docs/AboutUs.md

```diff
@@ -5,55 +5,16 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](https://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You can reach us at the email `e1290028@u.nus.edu`
```

Comment:
Maybe we can show our emails in a list, here's what I put down for this line:
```
You can reach us at the following emails:
- `youngseopark@u.nus.edu`
```

Reply:
@youngseopark05 I think that's a good idea!

Reply:
Fixed!